app/konnect/gateway-manager/data-plane-nodes/azure-peering.md

```diff
@@ -0,0 +1,89 @@
+---
+title: How to configure Azure VNET Peering
+---
+
+
+
+## How does VNET Peering Work
+Add a diagram
+
+In this diagram, the User Azure Cloud represents Azure Subscription you are running your microservices in, APIs, or applications. You can connect your infrastructure securely to {{site.konnect_short_name}} through Azure VNET Peering. On the Kong side, the Kong Azure Cloud is the azure subscription running your Dedicated Cloud Gateways, which ingests traffic coming in from the User VNET and securely exposes it to the internet.
+
+
+## Prerequisites
+
+* A {{site.konnect_short_name}} control plane
+* An Azure Tenant account with administrative privileges to create resources and manage peering, including the following information:
+  * Azure tenant ID
+  * Azure VNET subscription ID
+  * Azure VNET resource group name
+  * Azure VNET name
+
+
+## Configure VNET Peering
+
+### Configure Azure
```

Comment:
Is it okay to configure Azure first and then Konnect @prateekguptagh

Reply:
Yes, that can be done